src/Symfony/Component/Form/Form.php

```diff
@@ -749,7 +749,7 @@ public function isValid()
     /**
      * Returns the button that was used to submit the form.
      *
-     * @return ClickableInterface|null
+     * @return Button|ClickableInterface|null
```

Comment:
If I don't miss anything, we can remove the `@return` annotation entirely as the type can be deduced from the property.

Reply:
Thanks @xabbuh I don't mind doing this but it seems that the @return is used on all methods in the class. Should I keep to remain consistent?